tokio/src/process/mod.rs

```diff
@@ -690,6 +690,28 @@ impl Command {
         self
     }
 
+    /// Sets the process group ID (PGID) of the child process. Equivalent to a
+    /// setpgid call in the child process, but may be more efficient.
+    ///
+    /// Process groups determine which processes receive signals.
```

Comment:
This should document that it is unstable, how to enable unstable features (a link is fine - see doc for other unstable methods), and that it will become stable when our MSRV is new enough to allow this in a stable manner. It would also be nice to mention that you can do this today by constructing an `std::process::Command` and using our `From` impl to convert it.